sdcm/cluster.py

```diff
@@ -2266,15 +2266,18 @@ def get_scylla_version(self) -> str:
         if result.ok:
             self.scylla_version_detailed = result.stdout.strip()
             if build_id := self.get_scylla_build_id():
+                self.scylla_version = self.scylla_version_detailed
                 self.scylla_version_detailed += f" with build-id {build_id}"
                 self.log.info(f"Found ScyllaDB version with details: {self.scylla_version_detailed}")
+            self.log.debug(f"self.scylla_version_detailed={self.scylla_version_detailed}")
         else:
             if self.distro.is_rhel_like:
                 cmd = f"rpm --query --queryformat '%{{VERSION}}' {self.scylla_pkg()}"
             else:
                 cmd = f"dpkg-query --show --showformat '${{Version}}' {self.scylla_pkg()}"
             result = self.remoter.run(cmd, ignore_status=True)
 
+        cmd = ""
```

Comment:
why do you replace cmd with empty string?

Reply:
by mistake, removed.